### PR TITLE
Fix the _ga cookie parsing regression.

### LINKED ISSUE #9524: Regression: parsing _ga cookie incorrectly
Line [here](https://github.com/ampproject/amphtml/blob/master/src/service/url-replacements-impl.js#L259,) becomes incorrect after #9239 and #9499

### PR DESCRIPTION
**Pull request by lannka**
fixes #9524

created a TODO for an integration test: #9526

### CODE REVIEW

File: src/service/url-replacements-impl.js [hunk 1]
```diff
@@ -256,7 +256,8 @@ export class GlobalVariableSource extends VariableSource {
 
         // A temporary work around to extract Client ID from _ga cookie. #5761
         // TODO: replace with "filter" when it's in place. #2198
-        if (scope == '_ga') {
+        const cookieName = opt_cookieName || scope;
```
Comment (avimehta) [inline src/service/url-replacements-impl.js]: the value transformation will happen for scoped cid when loading from viewer. This won't have a real effect on data but is probably unintended. right? 

`CLIENT_ID(AMP_ECID_GOOGLE,, _ga)` from viewer should not result in a call to `extractClientIdFromGaCookie` but currently it does.  I think it is okay given that this code is temporary anyways..

**Comment by lannka** [comment]
added as a TODO. to capture this bug, we need an integration test.

**Comment by avimehta** [comment]
Ah. I see that you have created a bug for integration tests. LGTM from my side.

File: src/service/url-replacements-impl.js (hunk 1 above)
Comment (lannka) [inline src/service/url-replacements-impl.js]: well, it's not worth it to introduce "viewer" checking logic here. This is a no-op if it does not see GA formatted string.

as you said, it's really a temporary hack.

**Comment by lannka** [comment]
Also added a simple unit test.

**Pull request merged**